javascript/src/main/App.js

```diff
@@ -40,7 +41,11 @@ function App() {
           <Route path="/history/basic" exact component={Basic} />
           <Route path="/history/courseName" exact component={CourseName} />
           <Route path="/statistics" exact component={Statistics} />
+
+          <Route path="/statistics/classSize" exact component={ClassSize} />
```

Comment:
Just some unnecessary space here

Reply:
Good catch... But under the circumstances I can live with this :-)